### DISCUSSION TITLE
How can I view all projects even if they're not git repos?

### DISCUSSION THREAD

**Discussion by qodesmith** [alefragnani/vscode-project-manager#673] [Q&A] [answered]
I store all my projects in a flat folder structure like this:

```
/repos
  /project1
  /project2
  ...
```

Most of my projects are git repos, but some are not. How can I get all the projects in my `/repos` folder to show up in Project Manager? It seems like the only way right now is to:
1. Manually open the project up in VS Code
2. Add that project to the favorites list

**Comment by qodesmith** [comment, marked as answer]
Ok, I figured it out. For anyone else coming across this, here's how I did it:
1. Click the triple-dot menu on the top right of any of the Project Manager sections (such as Favorites) to open a drop down menu.
2. Click `Open Settings`.
3. Click the section labeled `All` on the left side of the settings window pane.
4. In the `Base Folders` input field, enter the root folder path that contains all your projects - for example `/User/qodesmith/repos`.
5. In the `Max Depth Recursion` input field, enter a value of `1` (this was what worked for me given my folder structure, adjust as needed).

Boom - now on the left side of VS Code when you have the Project Manager icon selected, there will be an `Any` collapsible section with all the folders listed.

**Reply by alefragnani** [reply]
Hi @qodesmith ,

You gotcha 👏 

If you want autodetected projects, use the feature you described. It works for Git, SVN, Mercurial and VSCode repos.

If you want projects with different names than its folder name, organize with tags, be remotes or multi-root workspaces, you should use **Favorites**.

Enjoy it
